packages/babel-plugin-transform-runtime/src/definitions.js

```diff
@@ -116,14 +123,18 @@ module.exports = {
       endsWith: "string/ends-with",
       fromCodePoint: "string/from-code-point",
       includes: "string/includes",
-      padLeft: "string/pad-left",
-      padRight: "string/pad-right",
+      padLeft: "string/pad-start", // deprecated
+      padRight: "string/pad-end", // deprecated
```

Comment:
Cool, I think these four files should to stay pointing at `pad-left` and `pad-right` (including `padStart` and `padEnd`) or else this could generate references to files that don't exist in older versions of `babel-runtime`. That may not be a strict requirement since I'd hope people have the versions in sync, but if we can avoid the chance of breaking changes, I'd be happier.